src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/UnlockableItemStack.java

```diff
@@ -0,0 +1,54 @@
+package io.github.thebusybiscuit.slimefun4.implementation.guide;
+
+import io.github.thebusybiscuit.slimefun4.core.researching.Research;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A subclass of {@link ItemStack} that indicates the item is unlockable in the guide
+ *
+ * @author RobotHanzo
+ */
+public class UnlockableItemStack extends ItemStack{
```

Comment:
Formatting. Also no need to expose this class since it is only for internal purposes.
```suggestion
class UnlockableItemStack extends ItemStack {
```